docs/fundamentals/build-like-google.md

```diff
@@ -0,0 +1,296 @@
+# Building like Google: Monorepos and Automation
+
+In this guide you will look at one of the most interesting parts of Nx. The part that can make so many things so much easier, that it can have a transformative effect on a team and even on an organization.
+
+What is it?
+
+It is a "monorepo" way of building applications.
+
+Working with multiple applications and libraries is difficult. From Google to Facebook, Uber, Twitter and more, a good amount of large software companies handle this challenge by taking a monorepo approach. And they have been doing so for years. These are some of the advantages this approach provides:
+
+- Everything at that current commit works together
+- Easy to split code into lib modules, easy to compose them
+- Easier dependency management
+- One build setup (Angular CLI)
+- Code editors and IDEs are "workspace" aware
+- Consistent developer experience
+
+To see how Nx delivers all of these, start with an empty Nx workspace:
+
+## Folder Structure
+
+```
+apps/
+libs/
+tools/
+angular.json
+nx.json
+package.json
+tsconfig.json
+tslint.json
+```
+
+An empty workspace has several root-level configuration files and the folders for applications, libraries, and tools. Since an Nx workspace is an Angular CLI workspace, most configuration files are created by Angular CLI. The only exception is `nx.json`, which contains Nx-specific configuration.
+
+## Applications and Libraries
+
+Nx supports two types of **projects**: applications from libraries.
+
+- An **application** is anything that can run in the browser or on the server. It's similar to a binary.
+- A **library** is a piece of code with a well-defined public API. A library can be imported into another library or application. You cannot run a library.
+
+### Applications
+
+Out of the box, Nx comes with two schematics for creating applications.
```

Comment:
The content is great but I think it's not fundamental to better understanding what an application is.

I feel like this should describe applications in more business terminology such as: _product, app, api, tool, project_